operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ResourceConfiguration.java

```diff
@@ -145,4 +145,12 @@ default Set<String> getEffectiveNamespaces(ConfigurationService configurationSer
   default Optional<ItemStore<R>> getItemStore() {
     return Optional.empty();
   }
+
+  /**
+   * The maximum amount of items to return for a single list call when starting an informer. If this
+   * is a not null it will result in paginating for the initial load of the informer cache.
+   */
+  default Long getInformerLimit() {
```

Comment:
Thinking if we could not find a more specific name. Like `getInformerListLimit` ?

Reply:
Otherwise LGTM

Reply:
I updated with your naming selection and added a link to the underlying method that ends up getting since the javadoc  there has more info.